src/PlusCommon/PlusCommon.h

```diff
@@ -59,6 +59,28 @@ enum PlusImagingMode
 ///////////////////////////////////////////////////////////////////
 // Logging
 
+class vtkPlusLogHelper
+{
+public:
+  double m_MinimumTimeBetweenLogging = 60.0;

```

Comment:
Convention is no m_, and to use this-> in .cxx files

Reply:
Is this only for public members? Otherwise e.g. `PlusTransformName::m_To` does not follow it.

Reply:
vtkPlus... classes are derived from vtkObject and use VTK conventions.
Non-VTK classes are names as Plus... and they use m_ prefix and no need for this->.

Reply:
Huh, did not know that. Cool

Reply:
Include unit in variable name to make sure it it interpreted correctly: For example: m_MinimumTimeBetweenLogging => m_MinimumTimeBetweenLoggingSec